### ISSUE TITLE
client/js/game.js RequireJS arguments order doesn't match callback function order

### ISSUE THREAD

**Issue by dachusa** [looperlands/looperlands#902]
Line 7 reads as:

```
             Item, Mob, Npc, Player, Character, Chest, Mobs, config, Exceptions, Fieldeffect, Float) {
```

should be 

```
             Item, Mob, Npc, Player, Character, Chest, Mobs, Exceptions, Fieldeffect, Config, Float) {
```

**Comment by dachusa** [comment]
Created pull request with the fix
[https://github.com/looperlands/looperlands/pull/903](https://github.com/looperlands/looperlands/pull/903)

**Comment by dachusa** [comment]
Confirmed Resolved with commit 7ba9ef58d7b9f66d01894324679e09ccd8d22e92
